### ISSUE TITLE
Performance Issues

### ISSUE THREAD

**Issue by jleandroperez** [Automattic/simplenote-macos#367]
### Details:
We're seeing reports on CPU spikes when typing. I've been tracking this down **for an extended journey** and it boils down to:

```
self.storage = [Storage newInstance];
[self.noteEditor.layoutManager replaceTextStorage:self.storage];
````

Using the default **NSTextStorage** instance results in no performance issues.

**Note:**
**Storage.swift** causes **SPTextView** drawRect method to lag about TWO orders of magnitude, when compared to the SDK's NSTextStorage.

**Sample note** that [triggers this available here](https://mc.a8c.com/pb/224b0/#plain).

H/T: **Thank you** @diegoreymendez for helping me out!!!

**Comment by KokkieH** [comment]
Noting we've had a report of this [via Twitter](https://twitter.com/rusmanarief/status/1164024960697810945)

**Comment by jacobrosenthal** [comment]
add me to the list. Its been a brutal few weeks.

**Comment by alxmjo** [comment]
Has this been fixed in 1.6.1?

**Comment by jleandroperez** [comment]
@alxmjo + @jacobrosenthal: 

Gentlemen, we've fixed a bug that caused the CPU usage to spike (#370), which affected High Sierra. 

We'll continue working on performance improvements alll over, if you *do* see a bottleneck (or freeze!), please don't hesitate to reach out.

**Thank you** for helping us improve Simplenote!

**Comment by alxmjo** [comment]
@jleandroperez Simplenote is working perfectly for me now. Thank you!

**Comment by jleandroperez** [comment]
Awesome, thanks for letting us know @alxmjo !!!

**Comment by jacobrosenthal** [comment]
Can confirm fix works for me. Thanks all

**Comment by jleandroperez** [comment]
Thank you @jacobrosenthal!! Hope you two, gentlemen, have an epic week!

**Comment by jleandroperez** [comment]
Closed via #368
